test/out_better_timestamp.rb

```diff
@@ -33,10 +32,14 @@ def test_remove_one_key
 
     mapped = {}
 
+    msec = 1
+    time = Time.now
```

Comment:
This seems to be using time when the plugin receives the log instead of using time stored as @timestamp.
I'm afraid this may generate unexpected value.

Reply:
Oops, I misunderstood this test case scenario.
I've fixed it.